Exercise_1.py

```diff
@@ -4,7 +4,16 @@
 # It returns location of x in given array arr  
 # if present, else returns -1 
 def binarySearch(arr, l, r, x): 
-  
+  mid = 0
+  while l < r:
+    mid = (l + r) // 2
```

Comment:
this is approach but Try to handle the integer overflow error to generalize for large data.